okhttp-tls/src/test/java/okhttp3/tls/HandshakeCertificatesTest.java

```diff
@@ -142,11 +142,11 @@
         .build();
     X509Certificate[] acceptedIssuers = handshakeCertificates.trustManager().getAcceptedIssuers();
     Set<String> names = Arrays.stream(acceptedIssuers)
-        .map(cert -> cert.getSubjectDN().getName().split("[ ,]")[0])
+        .map(cert -> cert.getSubjectDN().getName())
         .collect(Collectors.toCollection(LinkedHashSet::new));
 
     // It's safe to assume all platforms will have a major Internet certificate issuer.
-    assertThat(names).contains("CN=Entrust");
+    assertThat(names).anyMatch(s -> s.matches("[A-Z]+=Entrust.*"));
```

Comment:
bouncycastle is outputting OU first, but ordering also suggests current prefix logic is guaranteed to be stable.